backend/back/src/migrations/1698986506624-ChangeUserSettingsLanguageEnumAndAddAreaDefaultParameterReturnFlowValue.ts

```diff
@@ -0,0 +1,45 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class ChangeUserSettingsLanguageEnumAndAddAreaDefaultParameterReturnFlowValue1698986506624
+	implements MigrationInterface
+{
+	name = "ChangeUserSettingsLanguageEnumAndAddAreaDefaultParameterReturnFlowValue1698986506624";
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.query(
+			`ALTER TYPE "public"."user_settings_language_enum" RENAME TO "user_settings_language_enum_old"`,
+		);
+		await queryRunner.query(`CREATE TYPE "public"."user_settings_language_enum" AS ENUM('en', 'fr', 'is')`);
```

Comment:
so we will need to do a migration to add a language ?

Reply:
Exactly, that's really not an issue because typeorm generate them